docs/how_to_guides/configuring_data_contexts/how_to_use_environment_variables_or_a_yaml_file_to_populate_credentials.rst

```diff
@@ -0,0 +1,87 @@
+.. _how_to_guides__configuring_data_contexts__how_to_use_environment_variables_or_a_yaml_file_to_populate_credentials:
+
+How to Use Environment Variables or a YAML File to Populate Credentials
```

Comment:
```suggestion
How to use environment variables or a yaml file to populate credentials
```

Styling: We're using sentence case for titles. Please only capitalize the first word and other named concepts.

Reply:
Update: after discussing, we're going with YAML, with no backticks to make it a code block.